client/blocks/plugins-scheduled-updates/schedule-form.tsx

```diff
@@ -163,16 +162,12 @@ export const ScheduleForm = ( props: Props ) => {
 				} }
 			/>
 
-			{ isEnabled( 'plugins/multisite-scheduled-updates' ) && (
```

Comment:
do we need to remove the flag from config files?

Reply:
Not yet; we still use it in the router configuration, hiding the multisite schedule management.